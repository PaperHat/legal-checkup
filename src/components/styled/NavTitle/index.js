import styled from 'styled-components'
const NavTitle = styled.span`
  width: 156px;
  height: 54px;
  font-family: Avenir;
  font-size: 32px;
  font-weight: 900;
  font-style: normal;
  font-stretch: normal;
  line-height: 0.84;
  letter-spacing: normal;
  color: #8f8f8f;
  display: inline-block;
`

// const NavButton = styled.a`
//   color: red;
//   font-size: 16px;
//   border: none;
//   padding: 4px 10px;
//   background-color: rgb(219, 219, 212);
//   margin: 2px;
//   text-decoration: none;

//   &:hover {
//     color: white;
//     background-color: red;
//   }
// `
NavTitle.defaultName = 'StyledNavTitle'

export default NavTitle
