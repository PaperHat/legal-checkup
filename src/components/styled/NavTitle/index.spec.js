import React from 'react'
import { shallow } from 'enzyme'

import NavTitle from '.'

describe('components:styled:NavTitle', function () {
  it('renders the NavTitle and CSS properties properly', () =>
    expect(toJson(shallow(<NavTitle>NavButton</NavTitle>))).toMatchSnapshot())
})
