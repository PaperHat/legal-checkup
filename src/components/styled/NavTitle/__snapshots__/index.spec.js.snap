// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components:styled:NavTitle renders the NavTitle and CSS properties properly 1`] = `
.c0 {
  width: 156px;
  height: 54px;
  font-family: Avenir;
  font-size: 32px;
  font-weight: 900;
  font-style: normal;
  font-stretch: normal;
  line-height: 0.84;
  -webkit-letter-spacing: normal;
  -moz-letter-spacing: normal;
  -ms-letter-spacing: normal;
  letter-spacing: normal;
  color: #8f8f8f;
  display: inline-block;
  margin-top: 20px;
  margin-left: 10px;
  margin-bottom: 22px;
}

<span
  className="c0"
>
  NavButton
</span>
`;
